coretex/networking/network_manager_base.py

```diff
@@ -407,7 +410,9 @@ def formData(
             headers = self._headers("multipart/form-data")
             del headers["Content-Type"]
 
-            return self.request(endpoint, RequestType.post, headers, body = params, files = filesData)
+            timeout: Union[int, Tuple[int, int]] = (600, 300) if len(files) > 0 else REQUEST_TIMEOUT
```

Comment:
Can we extract the `timeout` type into a type alias since it is used in mutliple places?